### PR TITLE
Only allow RAMP intakes if feature toggle enabled

### PR DESCRIPTION
**Pull request by leikkisa**
connects #9397

### CODE REVIEW

File: app/models/request_issue.rb [hunk 1]
```diff
@@ -546,6 +546,8 @@ def create_decision_issues
       create_decision_issues_from_rating
     end
 
+    # We expect all rating request issues on an EP to get an associated rating created when they're decided
```
Comment (leikkisa) [inline app/models/request_issue.rb]: This comment is not really part of this PR, but I didn't want to make a separate PR just for this.

File: client/app/intake/pages/selectForm.jsx [hunk 2]
```diff
@@ -32,13 +33,25 @@ class SelectForm extends React.PureComponent {
...
+      }
+
+      { enableSearchableDropdown && <SearchableDropdown
         name="form-select"
         label="Which form are you processing?"
         placeholder="Enter or select form"
         options={radioOptions}
         onChange={this.setFormTypeFromDropdown}
         value={this.props.formType} />
+      }
```
Comment (leikkisa) [inline client/app/intake/pages/selectForm.jsx]: This was here before we took out the AMA feature toggle.  From the design team of the past: if we have more than 3 options, it should be a searchable dropdown, if not it should be radio buttons.

File: spec/support/intake_helpers.rb [hunk 3]
```diff
@@ -3,6 +3,19 @@
 # rubocop:disable Metrics/ModuleLength
 module IntakeHelpers
   # rubocop: disable Metrics/ParameterLists
+
+  def select_form(form_name)
```
Comment (pkarman) [inline spec/support/intake_helpers.rb]: I really like this refactor! Nice and elegant.

**Comment by Sjones352** [comment]
Passes UAT
